test/integration/create-next-app/templates-pages.test.ts

```diff
@@ -256,6 +256,11 @@ describe('create-next-app templates', () => {
         ],
         {
           cwd,
+          // @ts-expect-error process.env.NODE_ENV is not always avaliable in create-next-app
+          env: {
+            // Use NO_COLOR=1 to test picocolors
+            NO_COLOR: '1',
```

Comment:
```suggestion
            ...process.env,
            NO_COLOR: '1',
```

I think you need to preserve the existing env